### DISCUSSION TITLE
Restful api

### DISCUSSION THREAD

**Discussion by Sandeep06Dev** [vercel/next.js#15538] [Help] [answered]
Hey, The brilliant guys.

I have a restful api.. E.g. 
Http://localhost:3000/api/login

Currently i m using fetch to call it.. But the problem is my sensitive data can be easily show in client's browser network tab while i hit that rest api.. 

But as i seen, the API which i called inside getServerSideProps is not showing in client network tab.. Which is really awesome 😍❤️

Now, Is that any method or feature so i can call same api on button click without showing them to client network tab similar to  getServerSideProps  .?

**Comment by alanqchen** [comment, marked as answer]
The reason why the API request you send in `getServerSideProps` doesn't show in the network tab is because the request is made on the server, not on the client. Of course, the API request to login can only be made on the client side, so it will show up on the network tab. To keep the information secure, make sure that you are sending the requests using TLS/SSL in production.

**Reply by dhanushpani** [reply]
Is there a way where we can also hide  /api calls on network tab? Instead of using TLS/SSL?
